pkg/expr/dataplane.go

```diff
@@ -94,8 +95,20 @@ func handleDataplaneTimeseries(frames data.Frames) (mathexp.Results, error) {
 	}
 	res := mathexp.Results{}
 	res.Values = make([]mathexp.Value, 0, len(sc.Refs))
+
```

Comment:
graphite isn't dataplane yet, so it won't hit this path. In addition, don't want dataplane really doing anything here other then presenting the dataplane data - not manipulation.